Vagrantfile

```diff
@@ -0,0 +1,40 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  config.vm.provision "shell", privileged: true, inline: <<-SHELL
+    # Installing prerequisites for pyenv
+    apt-get update
+    apt-get -y install make build-essential libssl-dev zlib1g-dev \
```

Comment:
Really minor, but I'd be inclined to `sudo` these steps, and leave them inside the same unprivileged provisioning step for the sake of brevity